### PR TITLE
Add user and owner initial value to transition

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/back_office/models.py [hunk 1]
```diff
@@ -84,6 +84,28 @@ def _check_user_assigned(instances):
...
+    return None
+
+
+def autocomplete_owner_if_release_report(actions, objects):
+    try:
+        obj = objects[0]
+    except IndexError:
+        return None
+
+    if [True for i in actions if i.__name__ == 'release_report'] and obj.owner:
```
Comment (mkurek) [inline src/ralph/back_office/models.py]: `if obj.owner and any([a.__name__ == 'release_report' for a in actions]):`

File: src/ralph/back_office/models.py [hunk 2]
```diff
@@ -84,6 +84,32 @@ def _check_user_assigned(instances):
     return errors
 
 
+def autocomplete_user_if_release_report(actions, objects):
```
Comment (mkurek) [inline src/ralph/back_office/models.py]: extract this logic to single function where you'll pass attribute name (owner, user)

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**